### PR TITLE
feat: add themeprovider decoration

### LINKED ISSUE #294: Add ThemeProvider to Storybook

### PR DESCRIPTION
**Pull request by shanberg**
Closes #294 #295 

- Adds themeprovider decoration to storybook previews
- Adds 'Keen' and 'KeenDark' themes for demonstration purposes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "styled-components": "^3"
   },
   "dependencies": {
-    "polished": "^2.0.3"
+    "polished": "^2.0.3",
+    "storybook-addon-styled-component-theme": "^1.0.3"
```
Comment (kaitmore) [inline package.json]: I think this should be in dev deps

File: src/style/styleVariables.js [hunk 2]
```diff
@@ -62,6 +62,140 @@ export const FONT_WEIGHT_BASE = 400;
...
 
-
 // DEPRECATE
-export const DARK_ON_LIGHT_CONTRAST_ENHANCEMENT_RATIO = 2;
\ No newline at end of file
+export const DARK_ON_LIGHT_CONTRAST_ENHANCEMENT_RATIO = 2;
+
+export const keen = {
+  themeName: 'Keen',
+  name: 'Keen',
```
Comment (kaitmore) [inline src/style/styleVariables.js]: Why themeName and name?

**Comment by shanberg** [comment]
The broken StyleVariables story is interesting. Maybe without the themes addon controls, it’s unable to process the styles?

File: package.json (hunk 1 above)
Comment (shanberg) [reply package.json]: The broken StyleVariables story is interesting. Maybe without the themes addon controls, it’s unable to process the styles?

**Pull request merged**